students/grzegorz_bajorski/lesson_02_flow_control/leap_year.py

```diff
@@ -0,0 +1,8 @@
+print('Enter year number')
+year = int(input())
+
+if (year %4 ==0) and (year % 100 != 0) or (year % 400 == 0):
+	print('LEAF')
+else:
+	print('COMMON')
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.